safekeeper/src/http/routes.rs

```diff
@@ -125,6 +129,7 @@ async fn timeline_create_handler(mut request: Request<Body>) -> Result<Response<
         tenant_id: request_data.tenant_id,
         timeline_id: request_data.timeline_id,
     };
+    check_permission(&request, Some(zttid.tenant_id))?;
```

Comment:
This line is untested: `timeline_create_handler` is never called in the tests. Should we remove it, is it used at all?

Reply:
Discussed with @ars: it's a stub for a future code which will create timelines explicitly (which is probably better than implicit creation). So let's keep the code for now. We will test it when it's actually used.

Reply:
Discussed with @arssher: it's a stub for a future code which will create timelines explicitly (which is probably better than implicit creation). So let's keep the code for now. We will test it when it's actually used.